src/app.ts

```diff
@@ -0,0 +1,48 @@
+import Vue from 'vue';
+import { sync } from 'vuex-router-sync';
+import BootstrapVue from 'bootstrap-vue';
+import axios from 'axios';
+import { createProxy } from 'vuex-class-component';
+import App from './App.vue';
+import createRouter from './router';
+import createStore from './store/store';
+import InitGlobalComponents from '@/components/global';
+import UserStore from './store/user.vuex';
+import MobileStore from './store/mobile.vuex';
+import PageData from './store/page-data.vuex';
+
+Vue.use(BootstrapVue);
+
+InitGlobalComponents();
+
+Vue.config.productionTip = false;
+
+export default function createApp() {
+  const router = createRouter();
+  const store = createStore();
```

Comment:
Just to make sure I understand - Is this code ran on the same server instance for multiple instances of Vue?

Reply:
The server will import the frontend code once, but will call createApp once for each new connection, so it will have a fresh app, router, and store each time